contrib/postgresql-common/files/pg_versions

```diff
@@ -0,0 +1,225 @@
+#!/bin/sh
```

Comment:
this is crude and we don't want that, any "alternatives management" should be done through package management and not some weird haphazard scripts